packages/textlint/package.json

```diff
@@ -53,6 +53,7 @@
     "@textlint/feature-flag": "^3.1.2",
     "@textlint/fixer-formatter": "^3.1.4",
     "@textlint/kernel": "^3.1.5",
+    "@textlint/types": "^1.1.4",
```

Comment:
Previouly, `textlint` depended on `@textlint/types` implicitly.
It cause a bug like #599